hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -871,6 +869,11 @@
       'alert_handler.crashdump' : ['rstmgr_aon.alert_dump'],
       'csrng.entropy_src_hw_if' : ['entropy_src.entropy_src_hw_if'],
 
+      // Daisy chained LC RMA req/ack interface: LC -> Flash -> OTBN -> LC
```

Comment:
lgtm.

Reply:
it's too bad we can't do a last minute port name change and just make `lc_ctrl`'s `rma_req` and `rma_ack`... heh.

Reply:
Yes, indeed.

Good point about the chip-level test. I wasn't aware we already have one. Will look into this. Thanks @tjaychen !

Reply:
Just go confirm, I ran the `chip_sw_lc_walkthrough_rma` chip-level test and things seem to work as expected.

Reply:
Nice, this looks good to me.